wiki/Ranking_Criteria/osu!taiko/en.md

```diff
@@ -3,46 +3,161 @@ osu!taiko
 
 _Main page: [Ranking Criteria](/wiki/Ranking_Criteria)_
 
-These rules and guidelines has been discussed in the [osu!taiko Discussion Thread](https://osu.ppy.sh/forum/p/1275323). New rules or guidelines may be suggested in the [discussion forum](https://osu.ppy.sh/forum/87) which are going to be updated once the discussion reaches an agreement.
+These rules and guidelines have been discussed in the [Ranking Criteria subforum](https://osu.ppy.sh/community/forums/87). New rules or guidelines may be suggested in that forum and the criteria on this page will be updated once that discussion reaches an agreement.
```

Comment:
"new rules or guidelines" -> "new ones" to avoid repetition
"the criteria on this page" -> "this page" (unnecessary repetition)
"that" is grammatically incorrect here (doesn't refer to anything in particular), so I'd suggest replacing that with just "once discussions reach an agreement"